src/metaswitch/clearwater/config_manager/plugin_base.py

```diff
@@ -65,3 +71,11 @@ def on_config_changed(self, value, alarm):
         should also report the status of the controlled file to the alarm
         manager."""
         pass
+
+    @abstractmethod
+    def on_creating_etcd_key(self, value):
```

Comment:
Agreed. I've removed all of this again.
The shared config plugin will still check if we are attempting to write the default config, and if so, wont add the node to the restart queue